docs/learn/learn-safrole.md

```diff
@@ -0,0 +1,33 @@
+---
+id: learn-safrole
+title: "Polkadot Block Production: SAFROLE"
+sidebar_label: SAFROLE
+description: The Consensus Mechanism of Polkadot.
+keywords:
+  [
+    consensus,
+    proof of stake,
+    nominated proof of stake,
+    hybrid consensus,
+    block production,
+    sassafras,
+    safrole,
+    babe,
+  ]
+slug: ../learn-safrole
+---
+
+> This page will be updated as progress ensues.
```

Comment:
I would remove this. Non-necessary info for the reader, and usually we make sure the Wiki is up-to-date (more or less) ;) There are other instances in the Wiki and will add this as part of the wiki reorg